app/build.gradle

```diff
@@ -49,6 +49,11 @@ android {
             path file('src/main/jni/Android.mk')
         }
     }
+
+    packagingOptions {
+        pickFirst '**/*.so'
```

Comment:
I don't know what this pickFirst achieves. There are three so files built, for 4 architectures. This is on purpose Will this command only select the first? That will not work. Note the so files are only used for audio when the audio is not supported by your android device, or you select ffmpeg audio in the settings.

Reply:
I had the error "More than one file was found with OS independent path 'lib/x86/libavutil.so'" for every lib compiled with ffmpeg. 
It works for the emulator, I didn't test with a final apk. I'll try asap.

Reply:
These files are built in app/src/main/jni/ffmpeg/android-libs and copied to app/src/main/jnilibs . Perhaps the Mac OS is doing something different.

./arm64-v8a:
total 1576
-rwxr-xr-x 1 peter catch22 1126752 Mar  8 15:29 libavcodec.so
-rwxr-xr-x 1 peter catch22  395520 Mar  8 15:29 libavutil.so
-rwxr-xr-x 1 peter catch22   83568 Mar  8 15:29 libswresample.so

./armeabi-v7a:
total 1656
-rwxr-xr-x 1 peter catch22 1161404 Mar  8 15:28 libavcodec.so
-rwxr-xr-x 1 peter catch22  435944 Mar  8 15:28 libavutil.so
-rwxr-xr-x 1 peter catch22   91712 Mar  8 15:28 libswresample.so

./x86:
total 1684
-rwxr-xr-x 1 peter catch22 1194112 Mar  8 15:29 libavcodec.so
-rwxr-xr-x 1 peter catch22  435884 Mar  8 15:29 libavutil.so
-rwxr-xr-x 1 peter catch22   87556 Mar  8 15:29 libswresample.so

./x86_64:
total 1848
-rwxr-xr-x 1 peter catch22 1307320 Mar  8 15:29 libavcodec.so
-rwxr-xr-x 1 peter catch22  473544 Mar  8 15:29 libavutil.so
-rwxr-xr-x 1 peter catch22  104392 Mar  8 15:29 libswresample.so

Reply:
Yes, I had the same files but the build with `graddle` fails. Even when deleting `arm64-v8a armeabi-v7a x86_64` in order to avoid any choice, it still fails. I didn't find any other solution. But I'm new in android, I may have missed something.

Reply:
If you build the release version it does not even use x86. Maybe you could try changing the buildTypes in build.gradle. I have debug and release. Try create another one, like os2build with different ndk list, perhaps that will solve it.

Reply:
I just resync my branch with master and those options are not necessary anymore :muscle: . I have removed it.